test/unit/org/apache/cassandra/db/compaction/CancelCompactionsTest.java

```diff
@@ -465,7 +465,7 @@ public void testStandardCompactionTaskCancellation() throws Throwable
         createTable("create table %s (id int primary key, something int)");
         getCurrentColumnFamilyStore().disableAutoCompaction();
 
-        for (int i = 0; i < 10; i++)
+        for (int i = 0; i < 15; i++)
```

Comment:
We need slightly more SSTables for UCS in order to exceed the default threshold of SSTables for compaction. This value still works for STCS